en/components/general/angular-grid-overview-guide.md

```diff
@@ -6,16 +6,6 @@ _keywords: angular, angular app development, infragistics
 
 # A Complete Guide to Angular Grid and Angular App Development
 
```

Comment:
Please add here the following sentance - _Get to know the Angular Data Grid and how to use it [by checking out this informative section](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grids_and_lists.html#what-is-an-angular-data-grid) part of our Grid Overview topic._

Reply:
> Please add here the following sentance - _Get to know the Angular Data Grid and how to use it [by checking out this informative section](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grids_and_lists.html#what-is-an-angular-data-grid) part of our Grid Overview topic._

Applied :)